Src/ILGPU/Runtime/Cuda/CudaNativeMethods.cs

```diff
@@ -248,6 +248,37 @@ enum Peer2PeerAttribute
         CU_DEVICE_P2P_ATTRIBUTE_NATIVE_ATOMIC_SUPPORTED = 3
     }
 
+    [Flags]
+    public enum CudaEventFlags
+    {
+        /// <summary>
+        /// Default event creation flag.
+        /// </summary>
+        CU_EVENT_DEFAULT = 0,
+
+        /// <summary>
+        /// Specifies that the created event should use blocking synchronization. A CPU
```

Comment:
I am not sure if we are allowed to include the driver API documentation here... I would feel better if we just briefly summarize in one line roughly what the content is 🤓